api/v1alpha1/serviceintentions_webhook_test.go

```diff
@@ -398,3 +400,167 @@ func TestValidateServiceIntentions_Update(t *testing.T) {
 		})
 	}
 }
+
+// Test that we return patches to set Consul namespace fields to their defaults.
+// This test also tests OSS where we expect no patches since OSS has no
+// Consul namespaces.
+func TestHandle_ServiceIntentions_Patches(t *testing.T) {
```

Comment:
I added a new test rather than adding permutations to the existing tests because I thought that would make them too complicated.